buildings/gui/production_changes.py

```diff
@@ -210,6 +211,12 @@ def __init__(self, production_frame):
         iface.building_toolbar.show()
         self.disable_UI_functions()
 
+        self.production_frame.btn_circle.setEnabled(1)
+        self.production_frame.btn_circle.clicked.connect(self.setup_circle)
+        self.production_frame.btn_circle.show()
+        self.polyline = None
```

Comment:
The same as above. I think it's preferred to put the variables and signal connection to bulk_load_frame as the others do